### PR TITLE
Pass all test suites for RecipeBox and done with RecipeBox class

### PR DESCRIPTION
**Pull request by Alynn022**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor: Added all test suites for recipeBox class and finished class for recipeBox

How was this tested? 

- [ ] open index.html
- [ ] localhost
- [x] terminal

Where should the reviewer start - test: recipebox.js

Additional comments:

### CODE REVIEW

**Review by lschnell8 — APPROVED** [review]
This looks awesome Lynn! Looks like you figured out what I was going to share with ya'll about the sample data as well. Good job!

File: src/classes/RecipeBox.js [hunk 1]
```diff
@@ -23,19 +24,26 @@ class RecipeBox {
     })
     return matchingRecipes;
   }
-  findRecipeByIngredient(theUserInput) {
-    let matchingIngredientRecipes = [];
-    ingredientData.forEach((ingredient) => {
-      if(ingredient.name.includes(theUserInput)) {
-        matchingRecipes.push(recipe)
+  findIngredientById(theUserInput) {
```
Comment (StevieAmb) [inline src/classes/RecipeBox.js:27]: I wonder if we could talk through this function, again, please and thank you?

File: src/data/users.js [hunk 2]
```diff
@@ -6479,3 +6479,7 @@ const usersData = [
       ]
     }
   ]
+
```
Comment (StevieAmb) [inline src/data/users.js:6482]: When we import this, this will need curly braces around it.

**Review by StevieAmb — APPROVED** [review]

File: src/classes/RecipeBox.js (hunk 1 above)
Comment (Alynn022) [reply src/classes/RecipeBox.js:27]: Of course!

File: src/data/users.js (hunk 2 above)
Comment (Alynn022) [reply src/data/users.js:6482]: Got it !

**Pull request merged**